### PR TITLE
Added two files for poet2 integration with validator

### PR DESCRIPTION
**Pull request by abinsaml**
Signed-off-by: abinsaml <abinashX.samal@intel.com>

### CODE REVIEW

**Comment by rranjan3** [comment]
Create a separate dockerfile for integration. This should not checkout sawtooth-core. Instead make use of volumes to use the common checkout.

File: poet2-validator-integration.yaml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    # start the validator with an empty genesis batch
+    command: |
+      bash -c "
+        sawadm keygen --force && \
+        sawadm genesis && \
+        sawtooth-validator -vv \
+            --endpoint tcp://validator:8800 \
+            --bind component:tcp://eth0:4004 \
+            --bind network:tcp://eth0:8800 \
+            --bind consensus:tcp://eth0:5005 \
```
Comment (rranjan3) [inline poet2-validator-integration.yaml]: Update port to 5050 which would be the default one.

File: poet2-validator-integration.yaml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        - https_proxy
+        - no_proxy
+    volumes:
+      - ./:/project/sawtooth-poet2
+    working_dir: '/project/sawtooth-poet2/src/core'
+    container_name: poet-2-engine
+    depends_on:
+      - validator
+    entrypoint: "bash -c \"\
+        cargo run --bin poet2 -- -C tcp://validator:5005 --verbose \""
```
Comment (rranjan3) [inline poet2-validator-integration.yaml]: Update port to 5050.

Comment (abinsaml) [reply poet2-validator-integration.yaml]: Updated the port 5050 in poet2-validator-integration.yaml which would be default one.

File: src/core/src/consensus_state_store.rs [hunk 3]
```diff
@@ -0,0 +1,121 @@
+/*
```
Comment (askmish) [inline src/core/src/consensus_state_store.rs]: Consensus state store should've contained the functionalities for  storing consensus related info. See:  https://github.com/hyperledger/sawtooth-core/blob/8085604594105ccc478072cafbd1bafe1f4948d6/consensus/poet/core/sawtooth_poet/poet_consensus/consensus_state_store.py

Comment (askmish) [reply src/core/src/consensus_state_store.rs]: Please update the consensus state store to store consensus related info instead of blocks.

**Comment by askmish** [comment]
We're changing the code elsewhere. This PR is no longer relevant.

**Pull request closed without merging**